skills/declarative/calendarskill/dialogs/AcceptMeetingRequest/AcceptMeetingRequest.dialog

```diff
@@ -0,0 +1,79 @@
+{
+  "$kind": "Microsoft.AdaptiveDialog",
+  "$designer": {
+    "id": "jhrSvA",
+    "name": "AcceptMeetingRequest"
+  },
+  "autoEndDialog": "true",
+  "defaultResultProperty": "dialog.result",
+  "triggers": [
+    {
+      "$kind": "Microsoft.OnBeginDialog",
+      "$designer": {
+        "name": "BeginDialog",
+        "id": "Bxo20j"
+      },
+      "actions": [
+        {
+          "$kind": "Microsoft.BeginDialog",
+          "$designer": {
+            "id": "U2Iap6"
+          },
+          "dialog": "GetAuthToken"
+        },
+        {
+          "$kind": "Microsoft.TraceActivity",
+          "$designer": {
+            "id": "BdVEMc"
+          }
+        },
+        {
+          "$kind": "Microsoft.HttpRequest",
+          "$designer": {
+            "id": "XRHv2_"
+          },
+          "responseType": "Json",
+          "method": "POST",
+          "url": "https://graph.microsoft.com/v1.0/me/events/${dialog.meetingId}/accept",
+          "headers": {
+            "authorization": "${turn.token}"
+          },
+          "resultProperty": "dialog.api_result",
+          "contentType": "application/json"
+        },
+        {
+          "$kind": "Microsoft.IfCondition",
```

Comment:
I think the request dialog should return the result. The following logic should be handled in the parent dialog.

Reply:
Yes, it should return a result to make flow more robust. But I think it can just return the required data but not all the api result? For example True/False for delete/accept, MeetingRoomList for findmeetingroom.
We can discuss more offline.